string bond_id
---
bool is_recording
